packages/esm-patient-registration-app/src/patient-registration/input/basic-input/input/input.component.tsx

```diff
@@ -46,6 +46,7 @@ export const Input: React.FC<any> = ({ checkWarning, ...props }) => {
         <TextInput
           {...props}
           {...field}
+          style={{ height: '3rem' }}
```

Comment:
Having height of 3rem means the input is having size `lg`. We need the inputs to be size `md` (height: 2rem) which is by default. 

Try changing the size of the button instead. You can refer: https://react.carbondesignsystem.com/?path=/story/components-button--danger&args=tooltipPosition:bottom